docs/sources/writing-guide/reuse-shared-content.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Reuse shared content
+description: How to reuse chunks of content between pages
+keywords:
+  - content reuse
+  - shared content
+---
+
+# Reuse shared content
+
+Shared content is a sentence, paragraph, or whole page of text that makes sense in multiple contexts.
+Reusing shared content propagates changes from a single source file to one or more destination pages.
+
+This topic describes how to extract and share a chunk of content to multiple pages.
+
+## Before you begin
+
+- Identify a chunk of content that should be reused in multiple pages.
+
+## Steps
+
+To reuse shared content:
+
+1. Identify the sharing project.
+
+   If the shared content is to be reused in a single project, then the sharing project is that project.
+   However, if shared content is to be reused across multiple projects, for example Tempo and Grafana Enterprise Traces, then you must choose which project shares the content.
+
+   Prefer sharing from open source projects.
+
+1. In the sharing project, create the `docs/sources/shared/` directory if it does not exist.
+
+1. In the sharing project, create the file `docs/sources/shared/index.md` with the following contents:
+
+   ```markdown
+   ---
+   headless: true
+   ---
+   ```
+
+   The `index.md` file tells Hugo that the `docs/sources/shared` directory is a leaf bundle which is necessary for the `docs/shared` shortcode to access files stored within.
+
+   For more information about leaf bundles, refer to [Pages and page bundles]({{< relref "./documentation-structure/#pages-and-page-bundles" >}}).
+
+1. In the sharing project, create a file for the shared content in the `docs/sources/shared/` directory.
+
+   The file should be named to reflect its contents.
+
+   The file can be in a subdirectory, but that subdirectory must be in the `docs/sources/shared/` directory.
```

Comment:
we should insist on subdirectories within shared. it'll help keep the shared content more organized imo. maybe there's a reason not to do that?

Reply:
I think directory structure is a tough thing to provide general guidance on. It imposes a hierarchy on the contents that might not always be appropriate. I agree that using subdirectories absolutely can help with organization but it can also isolate pages and I am not sure I can explain the guidelines succinctly.